contracts/EnhancedStepFunctions.sol

```diff
@@ -160,61 +162,40 @@ contract ConversionRates2 is ConversionRates {
         return getImbalance(token, rateUpdateBlock, currentBlock);
```

Comment:
please rename this function.
maybe getImbalancePerToken.
of if you have better name.